R/plotFit.R

```diff
@@ -260,7 +260,7 @@
     paste0("ASSAY:   ", aenm, "\n\n",
            "NAME:    ", chnm, "\n",
            "CHID:    ", chid, spaces(8 - nchar(chid)),
-           "CASRN: ", casn, "\n", # spaces(16-nchar(casn)),"AGBY: ",agby,"\n",
```

Comment:
Did you want to delete the line above this instead? You've removed CASRN and have two lines for CHID